mi/dataset/parser/camhd_a.py

```diff
@@ -103,7 +107,7 @@ def parse_file(self):
                 utilities.formatted_timestamp_utc_time(file_datetime, TIMESTAMP_FORMAT))
 
             # Extract a particle and append it to the record buffer
-            particle = self._extract_sample(CamhdAInstrumentDataParticle, None, match.group('Path'),
+            particle = self._extract_sample(CamhdAInstrumentDataParticle, None, self._stream_handle.name,
```

Comment:
Instead of putting in the log file path, use the raw data server location of the mp4 file. This will make it easier for the UI to display location.